docs/cub/benchmarking.rst

```diff
@@ -181,7 +187,7 @@ We can then run the full benchmark suite from the build directory with:
 
 .. code-block:: bash
 
-    ../benchmarks/scripts/run.py
+    <root_dir_to_cccl>/cccl/benchmarks/scripts/run.py
```

Comment:
Remark: I wrote the documentation from the perspective of the cmake build directory. But I am not against the change, if this helps you (and users)!